### ISSUE TITLE
A11y - Update pagination nav's aria-label

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#204]
## Describe the task
The aria-label for the pagination nav is currently "pagination navigation" which announces on screenreaders redundantly - as "pagination navigation navigation."

## Acceptance Criteria
- [ ] Change the aria-label to just "pagination"

## Additional context
<img width="463" alt="Screenshot 2024-01-17 at 21 13 49" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/7329799/fccc2b88-71ea-44db-b8ad-b42092376a4f">

**Comment by marvieqa** [comment]
Note: we can omit the `role=navigation` because `<nav>` has an implicit role of navigation.

**Comment by markgoetz** [comment]
@bacitracin Can this issue be closed out now?

**Comment by bacitracin** [comment]
@markgoetz Yeah I think we're just behind on updating the board. I'll quickly check the ones you merged last week and close them out
